src/schematools/types.py

```diff
@@ -715,14 +715,14 @@ def __init__(
         self,
         *args: Any,
         parent_schema: DatasetSchema,
-        parent_table: Optional[DatasetTableSchema] = None,
+        _parent_table: Optional[DatasetTableSchema] = None,
```

Comment:
An argument with a leading `_` looks really strange to me.

Reply:
yes this was designed as an internal data-passing. It might need changes, but first I'd like to revert things that broke stuff.

Reply:
Ok.